src/components/Menu/Menu/Menu.tsx

```diff
@@ -28,7 +28,7 @@ import { getTestId } from "../../../tests/test-ids-utils";
 import { ComponentDefaultTestId } from "../../../tests/constants";
```

Comment:
I did not see any updates in the docs, let's think about it in our sync

Reply:
@hadasfa Can you plz remind me what's the context here? 🙏🏼